paymentapi/src/main/java/fi/hel/verkkokauppa/payment/paytrail/factory/PaytrailAuthClientFactory.java

```diff
@@ -10,20 +11,26 @@ public class PaytrailAuthClientFactory {
 	@Value("${paytrail.aggregate.merchant.id:}")
 	private String aggregateMerchantId;
 
-	@Value("${paytrail.merchant.secret:}")
-	private String secretKey;
+	@Value("${paytrail.aggregate.merchant.secret:}")
```

Comment:
Renamed paytrail.merchant.secret -> paytrial.aggregate.merchant.secret,  because it describes it better and it is the proper name from paytrail documentation.